resources/eventing/values.yaml

```diff
@@ -23,6 +23,21 @@ global:
     enabled: false
     # Storage type of the stream, memory or file.
     storage: memory
+    # Retention policy determines when messages should be deleted from the stream:
+    # (more info https://docs.nats.io/using-nats/developer/develop_jetstream/model_deep_dive#stream-limits-retention-and-policy):
+    # - interest: when all known observables have acknowledged a message it can be removed.
+    # - limits: messages are retained until any given limit is reached. This can be configured via maxMessages and maxBytes.
+    retentionPolicy: interest
```

Comment:
Any reason to keep these configs here? The only reason the above two configs were moved from the `controller` values.yaml to the `eventing` values.yaml was to share these configs between the controller and the nats charts.

Reply:
Done.